### DISCUSSION TITLE
How to remove the multi color bar from the top?

### DISCUSSION THREAD

**Discussion by pulkitkrishna00** [thuliteio/doks#1018] [Q&A] [answered]
How to remove the multi color bar from the top?

**Comment by h-enk** [comment, marked as answer]
By e.g. commenting out https://github.com/h-enk/doks/blob/master/layouts/partials/header/header.html#L9

**Reply by pulkitkrishna00** [reply]
And how to do the same for sidebar (on mobile view)?
